projects/stretch_objectnav/eval_episode.py

```diff
@@ -18,7 +17,6 @@
     rospy.init_node("eval_episode_stretch_objectnav")
     agent = ObjectNavAgent(config=config)
     env = StretchObjectNavEnv(config=config)
-    env.goto(STRETCH_HOME_Q)
```

Comment:
Why these changes in stretch obj nav?

Reply:
This is now in the `reset()` method of the Stretch environment

Reply:
sg